LoopKit/CarbKit/CarbMath.swift

```diff
@@ -490,12 +566,25 @@ fileprivate class CarbStatusBuilder<T: CarbEntry> {
         return min(entryGrams, max(minPredictedGrams, observedGrams))
     }
 
-    /// The maximum amount of time needed for the remaining entry grams to absorb, at the fixed minimum absorption rate
+    /// The amount of time needed for the remaining entry grams to absorb, at the currently estimated absorption rate
```

Comment:
For the non-linear model, the future amount of time is not necessarily a consistent rate, right? So this comment should not reference a single fixed rate.

Reply:
Agreed, will fix the comment